testsuite/features/secondary/allcli_software_channels.feature

```diff
@@ -80,7 +80,7 @@ Feature: Channel subscription via SSM
     And I wait until I do not see "Loading..." text
     Then radio button "openSUSE Leap 15.5 (x86_64)" should be checked
     And I wait until I do not see "Loading..." text
-    And I should see "Uyuni Proxy Devel for openSUSE Leap 15.5 (x86_64) (Development)" as unchecked
+    And I should see "Uyuni Proxy Stable for openSUSE Leap 15.5 (x86_64)" as unchecked
```

Comment:
Are you sure? "Stable" is what is released. But we should test development version, right?

Reply:
Btw: the name come from spacewalk-common-channels.ini and `Uyuni Proxy Devel for openSUSE Leap 15.5 (x86_64) (Development)` is still a valid name there

Reply:
Stable: https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/
Devel: https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/

I think when we want to run a test, we should test "Master"

Reply:
No I am not sure. @nodeg , do you have more information? Maybe we should have fixed this somehow else?

Reply:
Yes, it makes sense to test Master (`devel`) in the CI. We identified the issue and I will prepare a fix. So we can close this PR then.

Reply:
your PR is fine, closing this. Thanks a zillion @mcalmer for raising the alarm bell